README.md

```diff
@@ -2,6 +2,11 @@
 
 A FastAPI-based service that enables AI assistants to interact with Delta Lake tables stored in MinIO through Spark, implementing the Model Context Protocol (MCP) for natural language data operations.
 
+> **⚠️ Important Warning:** 
+> 
+> This service allows arbitrary `SELECT` queries to be executed against Delta Lake tables. Query results will be sent to the model host server, unless you are hosting your model locally.
```

Comment:
It allows arbitrary queries period, doesn't it, given the last PR? only read queries should work but if there's a misconfiguration that might not be true

Reply:
arbitrary query - forbidden keywords = arbitrary  select query

Reply:
assuming the forbidden keywords aren't missing any and assuming there's no way to get around the keywords somehow, but ok